src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -151,6 +151,10 @@ public void setPreviewInstructor(InstructorAttributes previewInstructor) {
     public void setRegisterMessage(String registerMessage) {
         this.registerMessage = registerMessage;
     }
+    
+    public void setSubmitAction(String submitAction) {
```

Comment:
Instead of a setter, can you have an abstract function that the child classes implement?